addons/more-costume-fonts/userscript.js

```diff
@@ -0,0 +1,63 @@
+export default async function ({ addon, console }) {
+	
+	const paper = await addon.tab.traps.getPaper();
+	const updateImage = paper.tool.onUpdateImage.bind('paper.tool');
+	const fonts = [
+		{ 'name': 'Sans Serif', 'family': 'Sans Serif' },
+		{ 'name': 'Serif', 'family': 'Serif' },
+		{ 'name': 'Handwriting', 'family': 'Handwriting' },
+		{ 'name': 'Marker', 'family': 'Marker' },
+		{ 'name': 'Curly', 'family': 'Curly' },
+		{ 'name': 'Pixel', 'family': 'Pixel' },
+		{ 'name': '中文', 'family': '"Microsoft YaHei", "微软雅黑", STXihei, "华文细黑"' },
+		{ 'name': '日本語', 'family': '"ヒラギノ角ゴ Pro W3", "Hiragino Kaku Gothic Pro", Osaka, "メイリオ", Meiryo, "ＭＳ Ｐゴシック", "MS PGothic"' },
+		{ 'name': '한국어', 'family': 'Malgun Gothic' },
+		{ 'name': 'Scratch', 'family': 'Scratch' },
+		{ 'name': 'Arial', 'family': 'Arial' },
+		{ 'name': 'Helvetica', 'family': 'Helvetica' },
+		{ 'name': 'Verdana', 'family': 'Verdana' },
+		{ 'name': 'Impact', 'family': 'Impact' },
+		{ 'name': 'Monospace', 'family': 'Monospace' }

```

Comment:
Are there any other fonts that work natively in the scratch editor? How can we find out?